core/src/main/java/hudson/model/queue/Tasks.java

```diff
@@ -26,14 +26,13 @@
 import hudson.model.Queue;
 import hudson.model.Queue.Item;
 import hudson.model.Queue.Task;
-import hudson.security.ACL;
 import javax.annotation.CheckForNull;
 import org.acegisecurity.Authentication;
 
 import java.util.Collection;
-import java.util.Collections;
 import javax.annotation.Nonnull;
 import jenkins.security.QueueItemAuthenticator;
+import jenkins.security.QueueItemAuthenticatorConfiguration;
```

Comment:
Unused import introduced by @oleg-nenashev apparently when resolving a merge conflict incorrectly. Harmless of course.